src/app/clusters/barrier-control-server/barrier-control-server.cpp

```diff
@@ -318,8 +318,8 @@ bool emberAfBarrierControlClusterBarrierControlGoToPercentCallback(
         state.currentPosition = getCurrentPosition(endpoint);
         state.targetPosition  = percentOpen;
         state.delayMs         = calculateDelayMs(endpoint, state.targetPosition, &state.increasing);
-        emberAfBarrierControlClusterPrintln("Scheduling barrier move from %d to %d with %dms delay", state.currentPosition,
-                                            state.targetPosition, state.delayMs);
+        emberAfBarrierControlClusterPrintln("Scheduling barrier move from %d to %d with " ChipLogFormatMEI "ms delay",
+                                            state.currentPosition, state.targetPosition, ChipLogValueMEI(state.delayMs));
```

Comment:
No, this is wrong.  `state.delayMs` is not a MEI; it's just a `uint32_t` and should be logged accordingly (probably as `"%" PRIu32`.

A MEI is a "data model identifier which allows manufacturer-scoping".  So cluster ids, attribute ids, command ids.  See Matter spec.